adaptive_navigation/example/lib/custom_scaffold.dart

```diff
@@ -8,6 +8,8 @@ class CustomScaffoldDemo extends StatefulWidget {
 
 class _CustomScaffoldDemoState extends State<CustomScaffoldDemo> {
   int _destinationCount = 5;
+  bool _fabInRail = false;
+  bool _overflowDestinationAddedToMenu = true;
```

Comment:
Rename to `_includeBaseDestinationInMenu` to match the rail

Reply:
DOne